src/CoreBluetooth/CBManager.cs

```diff
@@ -8,10 +8,12 @@
 namespace CoreBluetooth {
 	public partial class CBManager {
 
-#if !NET
-		[iOS (13,0), Watch (6,0)]
-#else
+#if NET
 		[SupportedOSPlatform ("ios13.0")]
+		[SupportedOSPlatform ("macos10.13")]
```

Comment:
Hmm did the tool add this one? Or is it a change that you added?

Reply:
Yes, because it is found on the definition version - https://github.com/chamons/xamarin-macios/blob/c0b28f207ccf9a2c2f1401314f6c25a96412bc06/src/corebluetooth.cs#L46